appveyor.yml

```diff
@@ -5,7 +5,7 @@ environment:
 
   global:
       PYTHON: "C:\\conda"
-      MINICONDA_VERSION: "3.5.5"
+      MINICONDA_VERSION: "latest"
```

Comment:
I worried a bit about that too, but the first step of each run is to do `conda update conda`, and one of the conda commits since 3.5.5 mentions fixing a pRoblem with conda updating itself.